contracts/optimistic-ethereum/OVM/verification/OVM_FraudVerifier.sol

```diff
@@ -249,4 +215,43 @@ contract OVM_FraudVerifier is Lib_AddressResolver, OVM_FraudContributor, iOVM_Fr
     {
         return address(getStateTransitioner(_preStateRoot, _txHash)) != address(0);
     }
+
+
+    /************************************
+     * Private Functions: Verification *
+     ************************************/
+
+    // NB: Stack too deep :/
+    function _deployTransitioner(bytes32 _preStateRoot, bytes32 _txHash) private {
```

Comment:
Would you mind just changing the structure of this signature to match the other signatures in this file (with each parameter on its own line, and `private` on its own line too) + add a function comment here? Thank you!